test/unit/custom_spec.js

```diff
@@ -107,3 +107,129 @@ describe("custom canvas rendering", function () {
       .catch(done.fail);
   });
 });
+
+describe("alternate document context", function () {
```

Comment:
Also, not that I look at this again a much better name would be `"custom ownerDocument"`, since that's actually consistent with the parameter name.